src/policy_pfrd.cpp

```diff
@@ -51,10 +56,9 @@ namespace pfrd
     fs::info_t info;
     const Branch *branch;
     const string *pfrdbasepath;
-    uint64_t cache_spaceavail_branches[256];
-    bool useable_branches[256];
-    const string *cache_basepath_branches[256];
     size_t branch_count = branches_.size();
+    bool useable_branches[branch_count];
```

Comment:
VLA, as I recall, isn't technically supported in C++ but shouldn't be a big deal so long as it works on gcc and clang. The bool array though... why not use the string ptr for that? Set to NULL at the top of the loop and then in the second loop if == NULL skip it.

Reply:
Also, can you please follow the existing styling? I realize it's non-standard and not currently consistent but I am in the process of fixing that.

Reply:
Yeah, good point!